properties.schema

```diff
@@ -208,6 +208,15 @@
                     }
                   }
                 },
+                "_suppressMarking": {
+                  "type":"boolean",
+                  "required":false,
+                  "title": "Suppress marking until assessment completion",
+                  "inputType": { "type": "Boolean", "options": [false, true]},
+                  "default": true,
+                  "validators": [],
+                  "help": "If enabled, individual question marking will only occur when the user has completed the assessment. Note: this is dependent on 'Show Marking' being enabled on the question component."
```

Comment:
I would change this to say 'will only occur when the user has used up all assessment attempts or has passed the assessment' - I think 'completed the assessment' could easily be misconstrued as 'got to the end of the assessment'.